app/views/ReportView/components/RapidSummary/index.tsx

```diff
@@ -193,10 +193,9 @@ const RapidSummary = ({
             const burdenResp = await api.get(`/reports/${report.ident}/mutation-burden`).request();
             if (burdenResp[0].qualitySvCount == null) {
               setPrimaryBurden(null);
-              console.log(burdenResp[0]);
             } else {
               setPrimaryBurden(burdenResp[0]);
-              console.log(burdenResp[0]);
+              console.log(burdenResp[0]); // For debug purpose
```

Comment:
should erase this from actual production code, probably forgot about this?

Reply:
that was my bad I will remove this right away